pkg/controller/loaders/fileLoader.go

```diff
@@ -21,6 +21,7 @@ type fileLoader struct {
 
 //FileLoader return loader from files
```

Comment:
I assume a loader is something that loads logs from some source?

Reply:
Yes, in provided lines we can get loader for file source.